public/tabs/chat/chat_page.tsx

```diff
@@ -122,7 +122,7 @@ export const ChatPage: React.FC<ChatPageProps> = (props) => {
         <EuiSpacer size="xs" />
         <ChatInputControls
           loading={chatState.llmResponding}
-          disabled={messagesLoading || chatState.llmResponding}
+          disabled={messagesLoading || conversationsLoading || chatState.llmResponding}
```

Comment:
Do not think it is good to disable ChatbotInput when conversations is loading. Perhaps we can update messagesLoading?

Reply:
Discussed offline, will revert this fix and leave it for a later PR